src/OctoshiftCLI.Tests/Octoshift/Services/GithubApiTests.cs

```diff
@@ -2782,6 +2789,30 @@ public async Task UpdateSecretScanningAlert_Calls_The_Right_Endpoint_With_Payloa
         _githubClientMock.Verify(m => m.PatchAsync(url, It.Is<object>(x => x.ToJson() == payload.ToJson()), null));
     }
 
+
+    [Fact]
+    public async Task UpdateSecretScanningAlert_Replaces_Null_Resolution_Comment_With_Empty_String()
```

Comment:
Based on my earlier comment we probably won't need this test.

Reply:
Agreed. 
And if we align with code scanning, we probably don't need that test either:  [UpdateCodeScanningAlert_Replaces_Null_Dismissed_Comment_With_Empty_String](https://github.com/github/gh-gei/blob/2fc08d1dcc025eac239487c38692c22657a41a27/src/OctoshiftCLI.Tests/Octoshift/Services/GithubApiTests.cs#L3118)

As per previous comment, happy to do a separate pull request to align both code scanning and secret scanning, or do the changes in this one.

Reply:
I have opened #1248 and added your suggestions there. Thanks again!
Happy to work on it next.